src/Core/Services/RequestValidator.cs

```diff
@@ -546,17 +546,17 @@ private static StoredProcedureDefinition TryGetStoredProcedureDefinition(string
         /// </summary>
```

Comment:
This summary should be updated since we no longer are checking that it can parse to a uint > 0.

Reply:
fixed.